### PR TITLE
Client dashboard - Language History TM-2290

### PR DESCRIPTION
**Pull request by scott062**
WIP - still need to remove fake data and integrate with WS endpoint. Haven't checked if we are already mapping the languages that are already coming through (also should double check that they STILL come through on client response). 

Possible merge conflcit with elizabeth's favorites update on client dashboard

### CODE REVIEW

File: src/Components/ProfileDashboard/ProfileDashboard.jsx [hunk 1]
```diff
@@ -120,46 +122,56 @@ const ProfileDashboard = ({
...
-                          <BidList
-                            bids={bidList}
-                            isPublic={isPublic}
-                            registerHandshake={registerHandshake}
-                            unregisterHandshake={unregisterHandshake}
-                            userId={userProfile.perdet_seq_number}
+                    isPublic &&
+                    <>
+                      <Column
+                        columns={columns[0]}
```
Comment (elizabeth-jimenez) [inline src/Components/ProfileDashboard/ProfileDashboard.jsx:128]: To fix this:
image

image

change to `columns[1]`

Comment (scott062) [reply src/Components/ProfileDashboard/ProfileDashboard.jsx]: thanks good catch

File: src/Components/ProfileDashboard/Languages/Languages.jsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+import SectionTitle from '../SectionTitle';
+import InformationDataPoint from '../InformationDataPoint';
+
+const Languages = props => {
+  const { languagesArray } = props;
+  const languagesArray$ = languagesArray || [];
+
+  const getTestDate = (langObj) => {
+    const testDate = get(langObj, 'test_date', false);
+    return isValid(new Date(testDate)) ? format(new Date(testDate), 'P') : '--/--/----';
```
Comment (elizabeth-jimenez) [inline src/Components/ProfileDashboard/Languages/Languages.jsx:13]: `new Date(false)` does return a valid Date 😅

Comment (scott062) [reply src/Components/ProfileDashboard/Languages/Languages.jsx:13]: RIP - fixed

**Review by elizabeth-jimenez — APPROVED** [review]
🍬 👍

**Pull request merged**